dspot-maven/src/main/java/fr/inria/diversify/dspot/maven/DSpotMojo.java

```diff
@@ -0,0 +1,171 @@
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+import fr.inria.diversify.utils.sosiefier.InputConfiguration;
+import fr.inria.stamp.Configuration;
+import fr.inria.stamp.JSAPOptions;
+import fr.inria.stamp.JSAPOptions.SelectorEnum;
+import fr.inria.stamp.Main;
+
+@Mojo(name = "mutationCoverage", defaultPhase = LifecyclePhase.VERIFY, requiresDependencyResolution = ResolutionScope.TEST)
+public class DSpotMojo extends AbstractMojo {
+
+	// Command Line parameters -> fr.inria.stamp.Configuration
+
+	private static final String BUILDER = "Maven";
+
+	@Parameter(defaultValue = "MethodAdd", property = "amplifiers")
+	private List<String> amplifiers;
+
+	@Parameter(defaultValue = "3", property = "iteration")
+	private Integer iteration;
+
+	@Parameter(defaultValue = "PitMutantScoreSelector", property = "test-criterion")
+	private String testCriterion;
+
+	@Parameter(defaultValue = "all", property = "test")
+	private List<String> namesOfTestCases;
+
+	@Parameter(defaultValue = "${project.build.directory}/dspot-report", property = "output-path")
+	private String outputPath;
+
+	@Parameter(defaultValue = "23", property = "randomSeed")
+	private Long randomSeed;
+
+	@Parameter(defaultValue = "10000", property = "timeOut")
+	private Integer timeOutInMs;
+
+	@Parameter(defaultValue = "PitMutantScoreSelector", property = "selector")
+	private String selector;
+
+	// Properties file parameters -> fr.inria.diversify.runner.InputConfiguration
+
+	@Parameter(defaultValue = "${project.basedir}", property = "project")
+	private File project;
+
+	@Parameter(defaultValue = "${project.build.sourceDirectory}", property = "src")
+	private File srcDir;
+
+	@Parameter(defaultValue = "${project.build.testSourceDirectory}", property = "test")
+	private File testDir;
+
+	@Parameter(defaultValue = "${project.build.outputDirectory}", property = "classes")
+	private File classesDir;
+
+	@Parameter(defaultValue = "${project.build.testOutputDirectory}", property = "testClasses")
+	private File testClassesDir;
+
+	@Parameter(defaultValue = "${project.build.directory}/tempDir", property = "tempDir")
+	private File tempDir;
+
+	// TODO: Command line option
+	@Parameter(defaultValue = "example.*", property = "filter")
+	private String filter;
+
+	@Parameter(defaultValue = "${env.M2_HOME}", property = "mavenHome")
+	private File mavenHome;
+
+	public void execute() throws MojoExecutionException, MojoFailureException {
+
+		Configuration configuration = new Configuration(
+				// path to file
+				null,
+				// Amplifiers
+				JSAPOptions.buildAmplifiersFromString(getAmplifiers().toArray(new String[0])),
+				// Iteration
+				getIteration(),
+				// testClases
+				getNamesOfTestCases(), getOutputPath().toString(), SelectorEnum.valueOf(getSelector()).buildSelector(),
```

Comment:
Here, there is twice `getNamesOfTestCases()`. I think we should have the Test Classes (a Class that contains Test Cases, and the Test Cases ( Test Cases are test method, annotated with **@Test**).